addon/components/yeti-table/body/row/cell/template.hbs

```diff
@@ -1,5 +1,5 @@
 {{#if column.visible}}
-  <td class="{{class}} {{column.columnClass}}" ...attributes>
+  <td class="{{class}} {{column.columnClass}} {{theme.tbodyCell}} " ...attributes>
```

Comment:
No need to add an extra space here in the end of the class attribute.

Reply:
fixed